libs/beacon/include/beacon/beacon_manager.hpp

```diff
@@ -108,6 +108,17 @@ class BeaconManager
   bool                           can_verify();
   std::string                    group_public_key() const;
   ///}
+  //
+
+private:
+  template <typename T, typename D>
+  friend struct serializers::MapSerializer;
+
+  // What the DKG should return
+  PrivateKey              secret_share_;       ///< Share of group private key (x_i)
```

Comment:
Why the move in the file?

Reply:
I wanted to demark the variables that are not metadata, thus if you want to ser/deser the class you must have at least these for it to fn correctly